src/components/QuotesApi.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+function QuotesApi() {
+  const [quotes, setQuotes] = useState([]);
+  const ApiKey = 'xyYovpvJWuK/3VYUQggBFQ==Ep1VlB5MOqrnejpl';
+  const category = 'happiness';
+  const fetchData = async () => {
+    await axios.get(`https://api.api-ninjas.com/v1/quotes?category=${category}`, {
+      headers: {
+        'x-api-key': ApiKey,
+      },
+    })
+      .then((response) => {
+        console.log(response);
+        setQuotes(response.data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
```

Comment:
You did a great job working on this milestone! 💯 However there are some things that could be improved! 🚀 

- [ ] According to the requirements:
![image](https://user-images.githubusercontent.com/77069011/228109887-1aaf513b-6b39-4f37-8a2f-b1e8c922b020.png)

You should show a loading and error state, but I'm unable to see any of them, please consider doing this! 🤓 🔧 

💡 You can have multiple `useState` to achieve this, here's an idea! 🤖 

```js
const [quotes, setQuotes] = useState([]);
const [loading, setloading] = useState(false);
const [error, setError] = useState(false);


// And then, in the JSX...
<main>
  { error && <div> Something went wrong </div> }
  { loading && <div> Loading, please wait... </div> }
  { !!quotes && <div> quotes.map ... </div>
</main>
```

- [ ] **[OPTIONAL]** I suggest that you use async/await instead of .then().catch(), here you're using both, please consider refactoring this! 🤓 🔧 

- [ ] **[OPTIONAL]** I suggest that you define your async function inside the `useEffect` hook to avoid possible side effects (like multiple rendering), you can read [this article](https://devtrium.com/posts/async-functions-useeffect) in order to know more about this! 🤓 🔧